apps/frontend/src/App.tsx

```diff
@@ -27,13 +29,15 @@ function AuthApp() {
     <BrowserRouter>
       <Routes>
         <Route path="/" element={<Layout children={<Landing />} />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/game/:gameId" element={<Layout children={<Game />} />} />
         <Route
-          path="/login"
-          element={<Login />}
+          path="/spectate/:gameId"
+          element={<Layout children={<Spectate />} />}
```

Comment:
this should ideally happen on the same url
We shouldnt expect spectators to come at a different url
fine for now tho